.devcontainer/scripts/configure-proxies.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
This file will be common for all repos - can we reuse same approach that MS used with https://raw.githubusercontent.com/devcontainers/features/main/src/common-utils/install.sh

Reply:
sure, but not in this PR :)